src/server/api/mutations/findNewCampaignContact.js

```diff
@@ -122,6 +123,12 @@ export const findNewCampaignContact = async (
       "assignedCount",
       updatedCount
     );
+
+    await telemetry.reportEvent("Assignment Dynamic", {
```

Comment:
this seems like it is going to be triggered a lot, imo it would be better as a metric rather than an event